src/connected/connections.ts

```diff
@@ -50,10 +50,10 @@ export class Connection extends VSCode.TreeItem {
     public readonly contextValue: 'sonarqubeConnection' | 'sonarcloudConnection',
     public status: ConnectionStatus
   ) {
-    super(label, VSCode.TreeItemCollapsibleState.Collapsed);
```

Comment:
Just a visual change to make all items in Connected Mode treeview expanded by default